sign_stage_rpms

```diff
@@ -14,7 +14,7 @@ for os in $OSES; do
 
     if [[ -n "$UNSIGNED_RPMS" ]]; then
       echo "$UNSIGNED_RPMS" | xargs ./sign_rpms
-      createrepo_c --update "$BASE/$arch"
+      createrepo_c --compress-type gz --database --update "$BASE/$arch"
```

Comment:
```
  --compress-type=COMPRESSION_TYPE             Which compression type to use for additional metadata files (comps, updateinfo, etc). Supported compressions are: bzip2, gzip, zck, zstd, xz.
  --general-compress-type=COMPRESSION_TYPE     Which compression type to use (even for primary, filelists and other xml). Supported compressions are: bzip2, gzip, zck, zstd, xz.
```

I *think* you want `--general-compress-type`? Did not try it out tho.

Reply:
hrm... yeah, I saw that earlier.  Switching